CMake/cdat_modules/pyparsing_external.cmake

```diff
@@ -2,5 +2,6 @@
 # create an external project to install MyProxyClient,
 # and configure and build it
 set(nm PYPARSING)
-
+set(OLD OFF)
```

Comment:
@aashish24 I think that bit here tells pipinstaller if we want it to use the `--old-and-unmanageable` option in setup.py which contrarily to what its name says make parallel build manageable...